hack/testdata/kubernetes-service.yaml

```diff
@@ -4,7 +4,6 @@ metadata:
   name: kubernetes
   namespace: default
 spec:
-  clusterIP: 10.0.0.1
```

Comment:
but this is the IP it $needs to be, no?  do we hit that before we hit 'resource exists' ?